{{cookiecutter.project_shortname}}/docker/wait-for-services.sh

```diff
@@ -27,30 +27,30 @@ check_ready "MySQL" _db_check
 _es_check(){ curl --output /dev/null --silent --head --fail http://localhost:9200 &>/dev/null; }
 check_ready "Elasticsearch" _es_check
 
-_kibana_check(){ curl --output /dev/null --silent --head --fail http://localhost:5601 &>/dev/null; }
-check_ready "Kibana" _kibana_check
-
 _redis_check(){ docker-compose exec cache bash -c 'redis-cli ping' | grep 'PONG' &> /dev/null; }
 check_ready "Redis" _redis_check
 
 _rabbit_check(){ docker-compose exec mq bash -c "rabbitmqctl status" &>/dev/null; }
 check_ready "RabbitMQ" _rabbit_check
 
-_web_server_check_css(){
-    string="Content-Type: text/css"
-    OUT=$(wget --spider -S -r -l 1 --page-requisites --no-check-certificate https://localhost 2>&1)
-    if echo $OUT | grep -q "$string"
-    then
-        return 0
-    else
-        return 1
-    fi
-}
+if [ "$1" == "--full" ]
+then
+    _web_server_check_css(){
```

Comment:
This function can still be defined outside the `if` block (basically I've no idea how `bash` functions work, so better be safe than sorry)